backend/mock/service/k8smock/k8smock.go

```diff
@@ -355,6 +355,10 @@ func (*svc) Clientsets(ctx context.Context) ([]string, error) {
 	return []string{"fake-user@fake-cluster"}, nil
 }
 
+func (*svc) GetK8sClientset(clientset string) (k8sservice.ContextClientset, error) {
```

Comment:
Was this just missing from a previous commit?

Reply:
Let me double check this actually

Reply:
Oh yeah sorry its been a few days since i looked at this code, this is the new public accessor method that i added to the k8s service interface, previously it was not possible to get at access a clientset before this pr.